spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -46,6 +46,32 @@
       )
     end
 
+    context 'ssn_is_unique attribute' do
+      context 'when the SSN is unique' do
+        it 'sets ssn_is_unique: true on the result' do
+          stub_vendor_requests
+          perform
+
+          result = document_capture_session.load_proofing_result[:result]
+          expect(result[:ssn_is_unique]).to be_truthy
```

Comment:
I think here we care about a literal `true` value. `be_truthy` could be anything, like a string "not true"
```suggestion
          expect(result[:ssn_is_unique]).to eq(true)
```